charts/vsphere-csi/v2.1.0/values.yaml

```diff
@@ -1,29 +1,65 @@
 vCenter:
   host: ""
-  datacenters: ""
   port: 443
   insecureFlag: true
+  clusterId: ""
+  datacenters: ""
   username: ""
   password: ""
   configSecret:
     name: "vsphere-config-secret"
     generate: true
-    configTemplate: |
-      [Global]
-      cluster-id = {{ uuidv4 | quote }}
-      user = {{ .Values.vCenter.username | quote }}
-      password = {{ .Values.vCenter.password | quote }}
-      port = {{ .Values.vCenter.port | quote }}
-      insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
+  configTemplate: |
```

Comment:
Why did we bump this out from the configSecret?

Reply:
reverted